packages/cli/src/lib/cmds/init.ts

```diff
@@ -36,10 +36,10 @@ async function confirmTwaConfig(twaManifest: TwaManifest, prompt: Prompt): Promi
   prompt.printMessage(messages.messageWebAppDetailsDesc);
 
   twaManifest.host = await prompt.promptInput(
-      messages.promptHostMessage, twaManifest.host, validateHost);
+      messages.promptHostMessage, twaManifest.host, await validateHost);
 
   twaManifest.startUrl = await prompt.promptInput(
-      messages.promptStartUrl, twaManifest.startUrl, createValidateString(1));
+      messages.promptStartUrl, twaManifest.startUrl, await createValidateString(1));
```

Comment:
`createValidateString` is not `async`. `await` is not needed here.
```suggestion
      messages.promptStartUrl, twaManifest.startUrl, createValidateString(1));
```